### DISCUSSION TITLE
In App Rating

### DISCUSSION THREAD

**Discussion by IbrahimSemab** [godotengine/godot-proposals#5949] [General Discussions]
Hi there, I am looking for a plugin or third-party add-ons that supports in app rating or Google Play In-App Reviews API in Godot.

**Comment by Calinou** [comment]
Please don't ask support questions here, as this discussion platform is meant to be used for feature discussions only. Use one of the other [community channels](https://godotengine.org/community) instead.

**Reply by IbrahimSemab** [reply]
Sorry, I don't knew.
